jekyll/_cci2/language-ruby.md

```diff
@@ -296,22 +302,12 @@ The `--profile` option reports the slowest examples of each run.
 
 For more on `circleci tests glob` and `circleci tests split` commands, please refer to our documentation on [Parallelism with CircleCI CLI](https://circleci.com/docs/2.0/parallelism-faster-jobs).
 
----
-
-Success! You just set up CircleCI 2.0 for a Ruby on Rails app. Check out our [Job page](https://circleci.com/gh/CircleCI-Public/circleci-demo-ruby-rails){:rel="nofollow"} to see how this looks when building on CircleCI.
-
-## Deployment
-
-See the [Deploy]({{ site.baseurl }}/2.0/deployment-integrations/) document for examples of deploy target configurations.
-
```

Comment:
This section was [also removed from the Android language guide](https://github.com/circleci/circleci-docs/pull/2550/commits/0ea331e5664c98d6d541110a53755fa074ca7d79). Removed here for consistency.